src/components/organisms/OrganismArticle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="mdc-card">
+    <div class="header">
+      <h1 class="mdc-typography--headline5 no-margin">{{ storage }}</h1>
+    </div>
+    <ul class="mdc-list">
+      <MoleculeArticleCard
+        class="mdc-list-item mdc-list-item--with-leading-image mdc-list-item--with-two-lines mdc-list-item--with-traili ng-icon"
+        v-for="i in 10"
+        :src="src"
+        :name="name"
+        :amount="amount"
+        :expiringIn="expiringIn"
+        :mhd="mhd"
+      ></MoleculeArticleCard>
```

Comment:
Code-Styling: remove closing html tag
```suggestion
      />
```